src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  background-color: white;
+}
+
+.nav__list__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+
+  transition: color 0.3s;
+}
+
+.nav__list__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  bottom: 0;
+  left: 0;
```

Comment:
It's unnecessary property

Reply:
content: "";
  bottom: 0;

Reply:
content: "";
  bottom: 0; - it's important (content - shows content and bottom: 0 - move line down)

Reply:
AAAAhhhh!!! How can we comment and show in comment only 1 line??? 
Yes, I mean only the last property - left: 0

Reply:
ok, I'm understand. I haven't done a review yet, so I don't know all the details. Thanks for review.